### DISCUSSION TITLE
Can I redirect the A to become a second Y?

### DISCUSSION THREAD

**Discussion by Timacaro** [phil-barrett/PicoCNC#25] [Q&A]
Good evening,

Before I purchase this card I would like to know if I can redirect the A to become a second Y?

Thank you for your response.

Romano

**Comment by phil-barrett** [comment]
If you mean use the board for a ganged dual motor gantry, the answer is yes.  You select Ganged Motors for the Y Axis.  Y becomes Y1 and A becomes Y2.  You can find [the web builder here](http://svn.io-engineering.com:8080/).

Also, you might want to look into [auto-squaring](https://www.grbl.org/single-post/it-s-hip-to-be-auto-square) to keep your gantry perpendicular to the Y Axis every time you home the machine.
